check-cf-space-quota-utilization/catalog.json

```diff
@@ -0,0 +1,632 @@
+            "repeat": null,
+            "errorMessages": [],
+            "dryRun": null
+          },
+          {
+            "execute": "applm-sapcp:ListCfApps:1",
+            "input": {
+              "resourceGroup": "$(.execution.input.spaceId)",
+              "password": "$(.execution.input.password)",
+              "pageSize": "100",
+              "region": "$(.execution.input.region)",
+              "user": "$(.execution.input.user)",
+              "subAccount": "$(.getCfOrg.output.body | toObject | .guid)",
+              "identityProvider": "$(.execution.input.identityProvider)"
+            },
+            "alias": "getCfApps",
+            "description": null,
+            "progressMessage": null,
+            "initialDelay": null,
+            "pause": null,
+            "when": null,
+            "validate": null,
+            "autoRetry": null,
+            "repeat": null,
+            "errorMessages": [],
+            "dryRun": null
+          },
+          {
+            "execute": "examples-<<<TENANT_ID>>>:GetCfAppsDetails:1",
+            "input": {
+              "defaultCfCredentials": "{\"user\": \"$(.execution.input.user)\", \"password\": \"$(.execution.input.password | toEscapedJson)\", \"resourceGroup\":\"$(.execution.input.spaceId)\", \"region\": \"$(.execution.input.region)\", \"subAccount\": \"$(.getCfOrg.output.body | toObject | .guid)\"}",
+              "appResources": "$(.getCfApps.output.resourceNames | map({ resourceName: . }))"
+            },
+            "alias": "calculateSpaceMemory",
+            "description": null,
+            "progressMessage": null,
+            "initialDelay": null,
+            "pause": null,
+            "when": null,
+            "validate": null,
+            "autoRetry": null,
+            "repeat": null,
+            "errorMessages": [],
+            "dryRun": null
+          },
+          {
+            "execute": "examples-<<<TENANT_ID>>>:CalculateSpaceMemPercentage:1",
+            "input": {
+              "totalSpaceMemory": "$(.getCfSpaceQuota.output.body | toObject | .apps.total_memory_in_mb)",
+              "usedSpaceMemory": "$(.calculateSpaceMemory.output.spaceMemory)"
+            },
+            "alias": "calculateSpaceMemoryPercentage",
+            "description": null,
+            "progressMessage": null,
+            "initialDelay": null,
+            "pause": null,
+            "when": null,
+            "validate": null,
+            "autoRetry": null,
+            "repeat": null,
+            "errorMessages": [],
+            "dryRun": null
+          },
+          {
+            "execute": "examples-<<<TENANT_ID>>>:CalculateOrgMemPercentage:1",
+            "input": {
+              "totalOrgMemory": "$(.getCfOrgQuota.output.body | toObject | .apps.total_memory_in_mb)",
+              "usedOrgMemory": "$(.getCfOrgUsage.output.body | toObject | .usage_summary.memory_in_mb)"
+            },
+            "alias": "calculateOrgMemoryPercentage",
```

Comment:
I think it's also possible to just use the Void command to make the calculation. This would allow you to delete the _CalculateOrgMemPercentage_ and _CalculateSpaceMemPercentage_ commands and simplify the whole scenario. Or is there another reason for these helper commands?